packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-email/main/ios/summaryofsteps.md

```diff
@@ -2,7 +2,7 @@ The following sequence diagram details each step in the authentication flow for
 
 <div class="full">
 
-![Displays a sequence diagram for the sign-in with passowrd and email use case](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-swift-sign-in-pwd-email.png)
+![Displays a sequence diagram of the sign-in with email and password flow](/img/oie-embedded-sdk/oie-embedded-swift-sign-in-pwd-email-flow-diagram.png)
```

Comment:
iOS Diagram comments:

1. Diagram missing resume() method call. See the code snippet to see how this works.
2. A callback happens using idx(client: didreceive) and the  stephandler delegate). This is not reflected in the diagram in all the steps. See the code snippet for more details.
3. The step names don't follow the new integration step conventions.

Reply:
Let's talk about this. I don't want to touch the iOS text at all because the code listing doesn't make sense to me and I'd rather someone else edit it into the same form as the rest of the guides.